### ISSUE TITLE
Is there a free method for automatic labeling  in YOLOv11 ?

### ISSUE THREAD

**Issue by anaduaa** [ultralytics/ultralytics#17265]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Is there a free method for automatic labeling once segmentation is completed in YOLOv11, as manual labeling takes a significant amount of time?

**Comment by Y-T-G** [comment]
You run prediction with `save_txt=True`.

https://docs.ultralytics.com/modes/predict/#inference-arguments

**Comment by Y-T-G** [comment]
The saved txt files can be imported to other annotation tools as labels.

**Comment by anaduaa** [comment]
For training step : if it is possible to do auto labeling for a new class  as we don't have any pre-trained model that trained on this class

**Comment by Y-T-G** [comment]
You can try YOLO World.

https://docs.ultralytics.com/models/yolo-world/

**Comment by danielsyahputra** [comment]
Hi @anaduaa  you can check out this code. In this repo I tried to integrate ultralytics with label-studio as a semi automatic labelling tools. Feel free to use that

**Comment by dantetemplar** [comment]
I would suggest using [CVAT](https://github.com/cvat-ai/cvat)

**Comment by glenn-jocher** [comment]
@dantetemplar thank you for the suggestion! CVAT is indeed a great tool for auto annotation and can be very helpful in streamlining the labeling process.

**Comment by Ela-Nova-Evo** [comment]
查看评论

**Comment by glenn-jocher** [comment]
@anaduaa you might also want to try Ultralytics' built-in [auto annotation utility](https://docs.ultralytics.com/usage/simple-utilities/#auto-labeling-annotations) which combines YOLO detection models with SAM for automatic segmentation labeling:

```python
from ultralytics.data.annotator import auto_annotate

auto_annotate(
    data="path/to/new/data",
    det_model="yolo11n.pt",
    sam_model="mobile_sam.pt",
    device="cuda",
    output_dir="path/to/save_labels",
)
```

This works well when you have a reasonable detection model and want to generate segmentation annotations automatically.
